#request
bool instatus
---
#response
bool outstatus